app/containers/MassEnergizeSuperAdmin/Community/Profile/About.js

```diff
@@ -56,16 +56,24 @@ class About extends React.Component {
 
 
   async getCSV(endpoint) {
-    console.log(this.props);
     const { community } = this.props;
     if (!community) {
       return;
     }
+
     let oldLoadingCSVs = this.state.loadingCSVs;
     this.setState({ loadingCSVs: oldLoadingCSVs.concat(endpoint) });
 
-    const csvResponse = await apiCallFile('/downloads.' + endpoint,
-      { community_id: community.id });
+    // csv downloads can be for a particular community, or they can be for all communities in which case the endpoint ends with '.all'.  
+    // In that case need to remove the '.all' from the endpoint to the API and not send the community ID in the body
+    const body = {};
+    const dotAll = endpoint.indexOf('.all');
+    if (dotAll > 0) {
+      endpoint = endpoint.substring(0,dotAll)
```

Comment:
thanks for adding this.  I will not press too much but i think the gymnastics going on here shows that we should be having to separate functions in the first place.  Determining what to call based on an endpoint is quite weird.